src/main/java/jalau/at18/battlecity/Missile.java

```diff
@@ -9,9 +9,9 @@ public class Missile {
     private int column1;
     private int row2;
     private int column2;
-    private Element[][] missileBoard;
-    private Missiles shotMissiles = new Missiles();
-    private Empty empty = new Empty();
+    private static Element[][] missileBoard;
```

Comment:
Isn't there a problem in the program? Initializing this variable in Static; because when I do this in my class, the class Battlecity didn't refresh the matrix...and the tank didn't move

Reply:
Thank you , i will look into it